app/models/organization.rb

```diff
@@ -11,6 +11,8 @@
 #  enable_child_based_requests    :boolean          default(TRUE), not null
 #  enable_individual_requests     :boolean          default(TRUE), not null
 #  enable_quantity_based_requests :boolean          default(TRUE), not null
+#  hide_package_column_on_receipt :boolean          default(FALSE)
+#  hide_value_columns_on_receipt  :boolean          default(FALSE)
```

Comment:
@cielf I'm kind of starting to feel like some of these fields belong in a "settings" model. E.g. these and `ytd_on_distribution_printout` - these are suuuper particular to be sitting on essentially our "god model". Thoughts?

Reply:
There is a convenience factor in having the organization view looking just at the organization model, I suppose.  I do agree that a lot of these are really really specific.

Reply:
Let's talk about it on Sunday -- it would be a new issue, in any case.

Reply:
Happy to refactor this PR to include a settings table, or print options table if that helps